modules/security-context-constraints-psa-synchronization.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: CONCEPT
+[id="security-context-constraints-psa-synchronization_{context}"]
+= Security context constraint synchronization with pod security standards
+
+In addition to the global pod security admission control configuration, {product-title} translates pod security profiles that use the `warn` and `audit` labels for every namespace.
+
+By using a controller, clusters examine `ServiceAccount` object permissions to apply security context constraints to each namespace. Security context constraints (SCCs) are mapped to pod security profiles based on their field values. Pod security admission control `warn` and `audit` labels are applied to the most privileged pod security profile in the namespace to prevent warnings and audit logging as pods are created. 
```

Comment:
Security Context Constraints are not applied to namespaces, their translation into a Pod Security profile is.

Reply:
```suggestion
By using a controller, clusters examine `ServiceAccount` object permissions to apply pod security profiles constraints to each namespace. Security context constraints (SCCs) are mapped to pod security profiles based on their field values; the controller uses these translated profiles. Pod security admission control `warn` and `audit` labels are applied to the most privileged pod security profile in the namespace to prevent warnings and audit logging as pods are created. 
```